src/main/java/com/easypost/model/beta/ReferralCustomerCollection.java

```diff
@@ -0,0 +1,46 @@
+package com.easypost.model.beta;
+
+import com.easypost.net.EasyPostResource;
+
+import java.util.List;
+
+public final class ReferralCustomerCollection extends EasyPostResource {
+    private List<ReferralCustomer> referralCustomers;
+    private Boolean hasMore;
```

Comment:
Nit: use lowercase (primitive) `boolean`
```suggestion
    private boolean hasMore;
```